fxmanifest.lua

```diff
@@ -1,34 +1,37 @@
+--[[ FX Information ]]--
 fx_version 'cerulean'
+use_experimental_fxv2_oal 'yes'
+lua54 'yes'
 game 'gta5'
 
-description 'QBX_VehicleKeys'
-repository 'https://github.com/Qbox-project/qbx_vehiclekeys'
+--[[ Resource Information ]]--
 version '1.0.0'
+license 'GPL-3.0-or-later'
+description 'vehicle key management system'
+repository 'https://github.com/Qbox-project/qbx_vehiclekeys'
+
+--[[ Manifest ]]--
+dependencies {
+    'ox_lib',
```

Comment:
This will make it so if you restart ox_lib, then this resource will be stopped. Is that the desired behavior?

Reply:
There are a few lib.callbacks in the script, so without ox_lib it may throw errors